Source/santad/EventProviders/SNTEndpointSecurityManager.mm

```diff
@@ -281,6 +328,17 @@ - (int)postAction:(santa_action_t)action forMessage:(santa_message_t)sm
   es_respond_result_t ret;
   switch (action) {
     case ACTION_RESPOND_ALLOW_COMPILER:
+      if (sm.pid >= PID_MAX) {
+        LOGE(@"Unable to watch compiler pid=%d >= pid_max=%d", sm.pid, PID_MAX);
+      } else {
+        LOGI(@"Watching compiler pid=%d path=%s", sm.pid, sm.path);
```

Comment:
This might be quite noisy, perhaps LOGD?

Reply:
Done